### ISSUE TITLE
Question: Theme without popups

### ISSUE THREAD

**Issue by justtobetrendy** [raml2html/default-theme#1]
Hi, we've done a theme based on this theme but removed the popups - they are to narrow and hard to follow when examples are large... how would you like us to contribute it ?

![apidoc](https://cloud.githubusercontent.com/assets/3965532/23317942/80c341e4-fa9e-11e6-9697-4efc761fb4ca.png)

**Comment by kevinrenskers** [comment]
Hey! The nice thing about the theme support with raml2html 5 is that you don't really need to do anything special. Just publish your theme on NPM, use the `raml2html-theme` keyword, and people will be able to find your theme and install it so they can use it with raml2html. 

So, you don't need to do anything special to contribute your theme, just publish it on NPM, open source it on Github (or wherever) and you're done. We don't need to review it, you retain full ownership, you're responsible for maintaining it. Hope that makes sense.

If you need help getting your theme onto NPM, please let me know. I'll write up a how-to at some point in the future, not sure when though. I'm closing this issue, but feel to reply here if you need help.
